docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -64,10 +60,32 @@ otherwise the {classanalysis} may not provide the best predictions. Read
 The ensemble algorithm that we use in the {stack} for {classification} is a type 
 of boosting called boosted tree regression model which combines multiple weak 
 models into a composite one. We use decision trees to learn to predict the 
-probability that a data point belongs to a certain class.
+probability that a data point belongs to a certain class. A sequence of decision 
+trees are trained and every decision tree learns from the mistakes of the 
+previous one. Every tree is an iteration of the last one, hence it improves the 
+decision made by the previous tree.
 //end::classification-algorithms[]
 
 
+[discrete]
+[[dfa-classification-performance]]
+==== {classification-cap} performance
+
+As a rule of thumb, a {classanalysis} with many classes takes more time to run 
+than a binary {classification} process when there are only two classes. The 
+relationship between the number of classes and the runtime is linear.
+
+The runtime scales approximately linearly with the number of involved documents 
+below 200.000 data points. Therefore, if you double the number of documents, 
+then the runtime of the analysis doubles respectively. Above 200.000 data 
+points, runtime increases faster.
+
+To achieve faster runtime, consider to set a smaller training percent, prepare 
```

Comment:
> To achieve faster runtime...

This sentence is similar to the previous sentence ("runtime increases faster"), but their meanings are different in a way that I find a bit confusing.  To avoid confusion, maybe we could change this sentence to something like:

> To improve the performance of your {classanalysis}....